war/src/main/less/base/style.less

```diff
@@ -219,9 +219,14 @@ td.tab {
 }
 
 pre {
-    white-space: pre-wrap;       /* css-3 */
-    word-wrap: break-word;       /* Internet Explorer 5.5+ */
-    margin: 0;
+  white-space: pre-wrap;
+  word-wrap: break-word;
+  margin: 0 0 2rem;
+  padding: 1rem;
+  border-radius: 10px;
+  color: white;
+  background-color: black;
```

Comment:
is this intentional?

![image](https://user-images.githubusercontent.com/21194782/152871084-5e78027f-26d7-4bfd-bb8d-c3681e62302b.png)

Reply:
Looks nice on dark theme:

![image](https://user-images.githubusercontent.com/21194782/152871989-06072eae-5bfa-4e6a-8cd5-a6dca739654a.png)

Reply:
Looks quite nice on dark theme for console output, but applying it on every `<pre>` looks weird:
![](https://i.imgur.com/fiTm9ja.png)

Reply:
May be okay on dark theme there, e.g script console page looks nice